Right-aligned prompts are not right-aligned in Bash/mintty on Windows

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### What happened?

The right-aligned portions of the prompt are never right-aligned; they sit somewhere in the middle of the screen. Moreover, they keep changing their alignment depending on the width of the left-aligned segments, which is weird.

![image](https://github.com/JanDeDobbeleer/oh-my-posh/assets/470626/3f6d811e-2435-46ca-80b6-e3a2af97e29f)

In case you are supposing this is a limitation with Bash/mintty, the same feature works fine in [Starship](https://starship.rs/).

### Theme

[powerlevel10k_rainbow](https://github.com/JanDeDobbeleer/oh-my-posh/blob/main/themes/powerlevel10k_rainbow.omp.json)

### What OS are you seeing the problem on?

Windows

### Which shell are you using?

bash

### Log output

```Shell
[DEBUG] 12:56:06.103 shell.go:Getenv:326 → mintty
[TRACE] 12:56:06.103 shell.go:Getenv(TERM_PROGRAM) - 519.2µs
[DEBUG] 12:56:06.103 shell.go:Getenv:326 → C:\Users\Bilge\AppData\Local
[TRACE] 12:56:06.103 shell.go:Getenv(LOCALAPPDATA) - 0s
[TRACE] 12:56:06.103 shell_windows.go:CachePath() - 0s
[DEBUG] 12:56:06.103 shell.go:Getenv:326 → C:/Users/Bilge/.p10k.omp.yaml
[TRACE] 12:56:06.103 shell.go:Getenv(POSH_THEME) - 0s
[DEBUG] 12:56:06.103 shell.go:Shell:601 → no shell name provided in flags, trying to detect it
[DEBUG] 12:56:06.109 shell.go:Shell:609 → process name: bash.exe
[TRACE] 12:56:06.109 shell.go:Shell() - 5.667ms
[DEBUG] 12:56:06.109 shell.go:resolveConfigPath:260 → Cygwin detected, using full path for config
[TRACE] 12:56:06.109 shell.go:resolveConfigPath() - 5.667ms
[TRACE] 12:56:06.109 shell.go:Init() - 6.1862ms
[TRACE] 12:56:06.109 shell.go:Flags() - 0s
[TRACE] 12:56:06.112 config.go:loadConfig() - 2.6119ms
[TRACE] 12:56:06.112 shell.go:Flags() - 0s
[DEBUG] 12:56:06.112 shell.go:Getenv:326 → NO DATA
[TRACE] 12:56:06.112 shell.go:Getenv(OMP_CACHE_DISABLED) - 0s
[TRACE] 12:56:06.112 shell_windows.go:WindowsRegistryKeyValue(HKEY_CURRENT_USER\Software\Microsoft\Windows\DWM\ColorizationColor) - 0s
[TRACE] 12:56:06.112 shell.go:GOOS() - 0s
[TRACE] 12:56:06.112 shell.go:GOOS() - 0s
[TRACE] 12:56:06.112 shell.go:GOOS() - 0s
[TRACE] 12:56:06.112 shell.go:GOOS() - 0s
[TRACE] 12:56:06.112 shell.go:GOOS() - 0s
[TRACE] 12:56:06.112 shell.go:GOOS() - 0s
[TRACE] 12:56:06.112 shell.go:GOOS() - 0s
[TRACE] 12:56:06.112 shell.go:GOOS() - 0s
[TRACE] 12:56:06.112 shell.go:GOOS() - 0s
[TRACE] 12:56:06.112 shell.go:GOOS() - 0s
[TRACE] 12:56:06.112 shell.go:GOOS() - 0s
[TRACE] 12:56:06.112 shell.go:GOOS() - 0s
[TRACE] 12:56:06.112 shell.go:GOOS() - 0s
[TRACE] 12:56:06.112 shell.go:GOOS() - 0s
[TRACE] 12:56:06.112 shell.go:GOOS() - 0s
[TRACE] 12:56:06.112 shell.go:GOOS() - 0s
[TRACE] 12:56:06.112 shell.go:GOOS() - 0s
[TRACE] 12:56:06.112 shell.go:GOOS() - 0s
[TRACE] 12:56:06.112 shell.go:GOOS() - 0s
[DEBUG] 12:56:06.112 shell_windows.go:WindowsRegistryKeyValue:217 → ColorizationColor(DWORD): 0xC40078D7
[TRACE] 12:56:06.112 shell.go:Shell() - 0s
[DEBUG] 12:56:06.112 shell.go:Getenv:326 → 5.2.26(1)-release
[TRACE] 12:56:06.112 shell.go:Getenv(POSH_SHELL_VERSION) - 0s
[DEBUG] 12:56:06.112 debug.go:PrintDebug:22 → Segment: Title
[DEBUG] 12:56:06.112 text.go:Render:72 → Rendering template: {{ .Shell }} in {{ .Folder }}
[TRACE] 12:56:06.112 shell_windows.go:Root() - 0s
[TRACE] 12:56:06.112 shell.go:Shell() - 0s
[TRACE] 12:56:06.112 shell.go:StatusCodes() - 0s
[TRACE] 12:56:06.112 shell_windows.go:IsWsl() - 0s
[DEBUG] 12:56:06.112 shell.go:TemplateCache:782 ↓
[TRACE] 12:56:06.112 shell.go:GOOS() - 0s
[DEBUG] 12:56:06.112 shell.go:Pwd:356 → C:\Users\Bilge\Documents\Adobe
[TRACE] 12:56:06.112 shell.go:Pwd() - 0s
[DEBUG] 12:56:06.112 shell_windows.go:58 → C:\Users\Bilge
[TRACE] 12:56:06.112 shell.go:GOOS() - 0s
[TRACE] 12:56:06.112 shell_windows.go:IsWsl() - 0s
[TRACE] 12:56:06.112 shell.go:GOOS() - 0s
[TRACE] 12:56:06.112 shell.go:GOOS() - 0s
[TRACE] 12:56:06.112 shell.go:GOOS() - 0s
[TRACE] 12:56:06.112 shell.go:GOOS() - 0s
[TRACE] 12:56:06.112 shell.go:GOOS() - 0s
[TRACE] 12:56:06.112 shell.go:GOOS() - 0s
[TRACE] 12:56:06.112 shell.go:GOOS() - 0s
[TRACE] 12:56:06.112 shell.go:GOOS() - 0s
[DEBUG] 12:56:06.112 shell.go:User:494 → Bilge
[TRACE] 12:56:06.112 shell.go:User() - 0s
[DEBUG] 12:56:06.112 shell.go:Host:511 → Fractal
[TRACE] 12:56:06.112 shell.go:Host() - 0s
[TRACE] 12:56:06.112 shell.go:GOOS() - 0s
[DEBUG] 12:56:06.112 shell.go:Getenv:326 → 1
[TRACE] 12:56:06.112 shell.go:Getenv(SHLVL) - 0s
[TRACE] 12:56:06.112 shell.go:TemplateCache() - 0s
[DEBUG] 12:56:06.112 shell.go:Getenv:326 → NO DATA
[TRACE] 12:56:06.112 shell.go:Getenv(POSH_CURSOR_LINE) - 0s
[DEBUG] 12:56:06.112 shell.go:Getenv:326 → NO DATA
[TRACE] 12:56:06.112 shell.go:Getenv(POSH_CURSOR_COLUMN) - 0s
[TRACE] 12:56:06.112 shell.go:Flags() - 0s
[TRACE] 12:56:06.113 shell.go:Flags() - 0s
[TRACE] 12:56:06.113 shell.go:Shell() - 0s
[TRACE] 12:56:06.113 shell.go:Shell() - 0s
[TRACE] 12:56:06.113 shell.go:Flags() - 0s
[TRACE] 12:56:06.113 shell.go:Flags() - 0s
[TRACE] 12:56:06.113 shell.go:Flags() - 0s
[TRACE] 12:56:06.113 shell.go:Pwd() - 0s
[TRACE] 12:56:06.113 shell.go:Pwd() - 0s
[TRACE] 12:56:06.113 shell.go:Pwd() - 0s
[DEBUG] 12:56:06.113 shell_windows.go:58 → C:\Users\Bilge
[TRACE] 12:56:06.113 shell.go:GOOS() - 0s
[DEBUG] 12:56:06.113 shell_windows.go:58 → C:\Users\Bilge
[TRACE] 12:56:06.113 shell.go:GOOS() - 0s
[DEBUG] 12:56:06.113 shell_windows.go:58 → C:\Users\Bilge
[TRACE] 12:56:06.113 shell.go:GOOS() - 0s
[DEBUG] 12:56:06.113 segment.go:SetEnabled:539 → Segment: Os
[DEBUG] 12:56:06.113 segment.go:SetEnabled:539 → Segment: Path
[TRACE] 12:56:06.113 shell.go:GOOS() - 0s
[TRACE] 12:56:06.113 shell.go:Pwd() - 0s
[TRACE] 12:56:06.113 shell.go:GOOS() - 0s
[TRACE] 12:56:06.113 shell.go:Shell() - 0s
[TRACE] 12:56:06.113 shell.go:Shell() - 0s
[DEBUG] 12:56:06.113 shell.go:Getenv:326 → C:\Users\Bilge\AppData\Local
[TRACE] 12:56:06.113 shell.go:Getenv(LOCALAPPDATA) - 0s
[DEBUG] 12:56:06.113 properties.go:GetString:28 → 
[TRACE] 12:56:06.113 shell.go:TemplateCache() - 0s
[DEBUG] 12:56:06.113 properties.go:GetBool:22 → mapped_locations_enabled: true
[DEBUG] 12:56:06.113 properties.go:GetString:28 → 
[DEBUG] 12:56:06.113 properties.go:GetString:28 → 
[DEBUG] 12:56:06.113 shell_windows.go:58 → C:\Users\Bilge
[TRACE] 12:56:06.113 shell.go:GOOS() - 0s
[TRACE] 12:56:06.113 shell.go:GOOS() - 0s
[DEBUG] 12:56:06.113 properties.go:GetString:28 → <nil>
[DEBUG] 12:56:06.113 properties.go:GetKeyValueMap:46 → mapped_locations: map[]
[TRACE] 12:56:06.113 shell.go:PathSeparator() - 0s
[TRACE] 12:56:06.113 shell.go:GOOS() - 0s
[TRACE] 12:56:06.113 shell.go:GOOS() - 0s
[TRACE] 12:56:06.113 shell.go:GOOS() - 0s
[TRACE] 12:56:06.113 shell.go:GOOS() - 0s
[TRACE] 12:56:06.113 shell.go:GOOS() - 0s
[TRACE] 12:56:06.113 shell.go:GOOS() - 0s
[TRACE] 12:56:06.113 shell.go:GOOS() - 0s
[TRACE] 12:56:06.113 shell.go:PathSeparator() - 0s
[TRACE] 12:56:06.113 shell.go:PathSeparator() - 0s
[TRACE] 12:56:06.113 shell.go:GOOS() - 0s
[TRACE] 12:56:06.113 shell.go:GOOS() - 0s
[TRACE] 12:56:06.113 shell.go:GOOS() - 0s
[TRACE] 12:56:06.113 shell.go:PathSeparator() - 0s
[TRACE] 12:56:06.113 shell.go:GOOS() - 0s
[TRACE] 12:56:06.113 shell.go:GOOS() - 0s
[TRACE] 12:56:06.113 shell.go:GOOS() - 0s
[TRACE] 12:56:06.113 shell.go:PathSeparator() - 0s
[TRACE] 12:56:06.113 shell.go:GOOS() - 0s
[TRACE] 12:56:06.113 shell.go:GOOS() - 0s
[TRACE] 12:56:06.113 shell.go:GOOS() - 0s
[TRACE] 12:56:06.113 shell.go:PathSeparator() - 0s
[TRACE] 12:56:06.113 shell.go:PathSeparator() - 0s
[TRACE] 12:56:06.113 shell.go:PathSeparator() - 0s
[DEBUG] 12:56:06.113 properties.go:GetString:28 → NO DATA
[TRACE] 12:56:06.113 shell.go:GOOS() - 0s
[TRACE] 12:56:06.113 shell.go:PathSeparator() - 0s
[TRACE] 12:56:06.113 shell.go:PathSeparator() - 0s
[TRACE] 12:56:06.113 shell.go:PathSeparator() - 0s
[DEBUG] 12:56:06.113 properties.go:GetString:28 → full
[DEBUG] 12:56:06.113 properties.go:GetStringArray:52 → cycle: []
[DEBUG] 12:56:06.113 properties.go:GetString:28 → NO DATA
[TRACE] 12:56:06.113 shell.go:PathSeparator() - 0s
[DEBUG] 12:56:06.113 properties.go:GetString:28 → \
[DEBUG] 12:56:06.113 properties.go:GetBool:22 → cycle_folder_separator: false
[DEBUG] 12:56:06.113 properties.go:GetString:28 → %s
[DEBUG] 12:56:06.113 properties.go:GetString:28 → %s
[DEBUG] 12:56:06.113 properties.go:GetString:28 → %s
[DEBUG] 12:56:06.113 properties.go:GetString:28 → %s
[TRACE] 12:56:06.113 shell.go:PathSeparator() - 0s
[TRACE] 12:56:06.113 shell.go:Pwd() - 0s
[TRACE] 12:56:06.113 shell.go:TemplateCache() - 0s
[TRACE] 12:56:06.113 shell.go:StackCount() - 0s
[DEBUG] 12:56:06.113 win32_windows.go:isWriteable:299 → not current user or in group
[DEBUG] 12:56:06.113 win32_windows.go:isWriteable:303 → current user is member of S-1-5-32-544
[DEBUG] 12:56:06.113 win32_windows.go:isWriteable:311 ↓
    WRITE_DAC
    WRITE_OWNER
    SYNCHRONIZE
    DELETE
    READ_CONTROL
[DEBUG] 12:56:06.113 win32_windows.go:isWriteable:313 → user has write access
[TRACE] 12:56:06.113 shell_windows.go:DirIsWritable() - 520µs
[TRACE] 12:56:06.113 shell.go:TemplateCache() - 0s
[DEBUG] 12:56:06.115 shell.go:CommandPath:553 → C:\Program Files\Git\mingw64\bin\git.exe
[TRACE] 12:56:06.115 shell.go:CommandPath(git.exe) - 2.5081ms
[TRACE] 12:56:06.115 shell.go:HasCommand(git.exe) - 2.5081ms
[TRACE] 12:56:06.115 shell.go:Pwd() - 0s
[ERROR] 12:56:06.115 shell.go:HasParentFilePath:697 → CreateFile .git: The system cannot find the file specified.
[TRACE] 12:56:06.115 shell.go:HasParentFilePath(.git) - 0s
[DEBUG] 12:56:06.115 properties.go:GetBool:22 → fetch_bare_info: false
[DEBUG] 12:56:06.115 text.go:Render:72 → Rendering template:   {{ .Path }}
[DEBUG] 12:56:06.115 text.go:Render:72 → Rendering template:  {{ if .WSL }}WSL at {{ end }}{{.Icon}}
[TRACE] 12:56:06.115 shell.go:TemplateCache() - 0s
[TRACE] 12:56:06.115 shell.go:TemplateCache() - 0s
[TRACE] 12:56:06.115 shell.go:Shell() - 0s
[TRACE] 12:56:06.115 shell.go:Shell() - 0s
[DEBUG] 12:56:06.115 text.go:Render:72 → Rendering template: diamond
[DEBUG] 12:56:06.115 text.go:Render:72 → Rendering template: powerline
[DEBUG] 12:56:06.115 text.go:Render:72 → Rendering template: powerline
[TRACE] 12:56:06.115 shell.go:Shell() - 0s
[TRACE] 12:56:06.115 shell.go:Shell() - 0s
[TRACE] 12:56:06.115 shell.go:Shell() - 0s
[TRACE] 12:56:06.115 shell.go:Shell() - 0s
[TRACE] 12:56:06.115 shell.go:Flags() - 0s
[TRACE] 12:56:06.115 shell.go:Flags() - 0s
[TRACE] 12:56:06.115 shell.go:Flags() - 0s
[TRACE] 12:56:06.115 shell.go:Pwd() - 0s
[TRACE] 12:56:06.115 shell.go:Pwd() - 0s
[DEBUG] 12:56:06.115 shell_windows.go:58 → C:\Users\Bilge
[DEBUG] 12:56:06.115 shell_windows.go:58 → C:\Users\Bilge
[TRACE] 12:56:06.115 shell.go:GOOS() - 0s
[TRACE] 12:56:06.115 shell.go:GOOS() - 0s
[DEBUG] 12:56:06.115 segment.go:SetEnabled:539 → Segment: Time
[DEBUG] 12:56:06.115 segment.go:SetEnabled:539 → Segment: Go
[TRACE] 12:56:06.115 shell.go:Flags() - 0s
[TRACE] 12:56:06.115 shell.go:Flags() - 0s
[TRACE] 12:56:06.115 shell.go:Pwd() - 0s
[DEBUG] 12:56:06.115 properties.go:GetString:28 → files
[DEBUG] 12:56:06.115 properties.go:GetStringArray:52 → extensions: [*.go go.mod]
[TRACE] 12:56:06.115 shell.go:Flags() - 0s
[DEBUG] 12:56:06.115 properties.go:GetStringArray:52 → folders: []
[DEBUG] 12:56:06.115 shell_windows.go:58 → C:\Users\Bilge
[DEBUG] 12:56:06.115 properties.go:GetBool:22 → home_enabled: false
[TRACE] 12:56:06.115 shell.go:GOOS() - 0s
[TRACE] 12:56:06.115 shell.go:Pwd() - 0s
[DEBUG] 12:56:06.115 segment.go:SetEnabled:539 → Segment: Aws
[DEBUG] 12:56:06.115 shell_windows.go:58 → C:\Users\Bilge
[DEBUG] 12:56:06.115 properties.go:GetString:28 → files
[TRACE] 12:56:06.115 shell.go:Pwd() - 0s
[TRACE] 12:56:06.115 shell.go:Pwd() - 0s
[DEBUG] 12:56:06.115 properties.go:GetBool:22 → display_default: false
[DEBUG] 12:56:06.115 shell.go:Getenv:326 → NO DATA
[TRACE] 12:56:06.115 shell.go:Pwd() - 0s
[TRACE] 12:56:06.115 shell.go:Getenv(AWS_VAULT) - 0s
[DEBUG] 12:56:06.115 segment.go:SetEnabled:539 → Segment: Python
[DEBUG] 12:56:06.115 shell.go:Getenv:326 → NO DATA
[TRACE] 12:56:06.115 shell.go:Getenv(AWS_DEFAULT_PROFILE) - 0s
[DEBUG] 12:56:06.115 shell_windows.go:58 → C:\Users\Bilge
[DEBUG] 12:56:06.115 properties.go:GetStringArray:52 → extensions: [*.py *.ipynb pyproject.toml venv.bak]
[DEBUG] 12:56:06.115 properties.go:GetStringArray:52 → folders: [.venv venv virtualenv venv-win pyenv-win]
[DEBUG] 12:56:06.115 shell.go:Getenv:326 → NO DATA
[TRACE] 12:56:06.115 shell.go:Getenv(AWS_PROFILE) - 0s
[DEBUG] 12:56:06.115 properties.go:GetBool:22 → home_enabled: false
[DEBUG] 12:56:06.115 shell.go:Getenv:326 → NO DATA
[TRACE] 12:56:06.115 shell.go:Pwd() - 0s
[TRACE] 12:56:06.115 shell.go:Pwd() - 0s
[TRACE] 12:56:06.115 shell.go:Getenv(AWS_REGION) - 0s
[DEBUG] 12:56:06.115 shell_windows.go:58 → C:\Users\Bilge
[DEBUG] 12:56:06.115 shell.go:Getenv:326 → NO DATA
[TRACE] 12:56:06.115 shell.go:Getenv(AWS_DEFAULT_REGION) - 0s
[DEBUG] 12:56:06.115 properties.go:GetBool:22 → fetch_virtual_env: false
[TRACE] 12:56:06.115 shell.go:GOOS() - 0s
[DEBUG] 12:56:06.115 shell.go:Getenv:326 → NO DATA
[TRACE] 12:56:06.115 shell.go:Pwd() - 0s
[DEBUG] 12:56:06.115 properties.go:GetString:28 → 15:04:05
[TRACE] 12:56:06.115 shell.go:Getenv(AWS_CONFIG_FILE) - 0s
[DEBUG] 12:56:06.115 shell_windows.go:58 → C:\Users\Bilge
[TRACE] 12:56:06.115 shell.go:TemplateCache() - 0s
[DEBUG] 12:56:06.115 shell_windows.go:58 → C:\Users\Bilge
[DEBUG] 12:56:06.115 segment.go:SetEnabled:539 → Segment: Executiontime
[TRACE] 12:56:06.115 shell.go:GOOS() - 0s
[TRACE] 12:56:06.115 shell.go:Flags() - 0s
[DEBUG] 12:56:06.115 segment.go:SetEnabled:539 → Segment: Julia
[TRACE] 12:56:06.115 shell.go:ExecutionTime() - 0s
[DEBUG] 12:56:06.115 properties.go:GetStringArray:52 → extensions: [*.jl]
[TRACE] 12:56:06.115 shell.go:Flags() - 0s
[DEBUG] 12:56:06.115 shell.go:HasFilesInDir:409 → false
[DEBUG] 12:56:06.115 properties.go:GetFloat64:34 → threshold: 500.000000
[DEBUG] 12:56:06.115 properties.go:GetStringArray:52 → folders: []
[TRACE] 12:56:06.115 shell.go:HasFilesInDir(*.go) - 0s
[DEBUG] 12:56:06.115 properties.go:GetBool:22 → home_enabled: false
[TRACE] 12:56:06.115 shell.go:Flags() - 0s
[TRACE] 12:56:06.115 shell.go:Pwd() - 0s
[TRACE] 12:56:06.115 shell.go:Pwd() - 0s
[TRACE] 12:56:06.115 shell.go:Pwd() - 0s
[DEBUG] 12:56:06.115 shell.go:HasFilesInDir:409 → false
[DEBUG] 12:56:06.115 shell_windows.go:58 → C:\Users\Bilge
[TRACE] 12:56:06.115 shell.go:HasFilesInDir(go.mod) - 0s
[DEBUG] 12:56:06.115 shell_windows.go:58 → C:\Users\Bilge
[DEBUG] 12:56:06.115 properties.go:GetString:28 → files
[TRACE] 12:56:06.115 shell.go:GOOS() - 0s
[TRACE] 12:56:06.115 shell.go:Pwd() - 0s
[TRACE] 12:56:06.115 shell.go:Flags() - 0s
[TRACE] 12:56:06.115 shell.go:Pwd() - 0s
[DEBUG] 12:56:06.115 segment.go:SetEnabled:539 → Segment: Root
[DEBUG] 12:56:06.115 shell.go:HasFilesInDir:409 → false
[DEBUG] 12:56:06.115 shell_windows.go:58 → C:\Users\Bilge
[TRACE] 12:56:06.115 shell.go:HasFilesInDir(*.jl) - 0s
[TRACE] 12:56:06.115 shell.go:GOOS() - 0s
[DEBUG] 12:56:06.115 segment.go:SetEnabled:539 → Segment: Ruby
[TRACE] 12:56:06.115 shell.go:Pwd() - 0s
[DEBUG] 12:56:06.115 shell_windows.go:58 → C:\Users\Bilge
[DEBUG] 12:56:06.115 properties.go:GetStringArray:52 → extensions: [*.rb Rakefile Gemfile]
[DEBUG] 12:56:06.115 properties.go:GetString:28 → {{ .Code }}
[TRACE] 12:56:06.115 shell.go:GOOS() - 0s
[DEBUG] 12:56:06.115 properties.go:GetStringArray:52 → folders: []
[DEBUG] 12:56:06.115 segment.go:SetEnabled:539 → Segment: Azfunc
[DEBUG] 12:56:06.115 properties.go:GetBool:22 → home_enabled: false
[TRACE] 12:56:06.115 shell.go:Pwd() - 0s
[TRACE] 12:56:06.115 shell.go:Pwd() - 0s
[DEBUG] 12:56:06.115 shell_windows.go:58 → C:\Users\Bilge
[DEBUG] 12:56:06.115 properties.go:GetStringArray:52 → folders: []
[DEBUG] 12:56:06.115 properties.go:GetString:28 → files
[DEBUG] 12:56:06.115 properties.go:GetBool:22 → home_enabled: false
[DEBUG] 12:56:06.115 shell_windows.go:58 → C:\Users\Bilge
[TRACE] 12:56:06.115 shell.go:Pwd() - 0s
[TRACE] 12:56:06.115 shell.go:Pwd() - 0s
[TRACE] 12:56:06.115 shell.go:GOOS() - 0s
[DEBUG] 12:56:06.115 shell_windows.go:58 → C:\Users\Bilge
[DEBUG] 12:56:06.115 shell.go:HasFilesInDir:409 → false
[DEBUG] 12:56:06.115 properties.go:GetString:28 → files
[TRACE] 12:56:06.115 shell.go:HasFilesInDir(*.rb) - 0s
[DEBUG] 12:56:06.115 segment.go:SetEnabled:539 → Segment: Status
[TRACE] 12:56:06.115 shell.go:Pwd() - 0s
[TRACE] 12:56:06.115 shell.go:Pwd() - 0s
[DEBUG] 12:56:06.115 shell.go:HasFilesInDir:409 → false
[TRACE] 12:56:06.115 shell.go:StatusCodes() - 0s
[DEBUG] 12:56:06.115 shell.go:HasFilesInDir:409 → false
[DEBUG] 12:56:06.115 text.go:Render:72 → Rendering template: {{ .Code }}
[TRACE] 12:56:06.115 shell.go:HasFilesInDir(Rakefile) - 0s
[TRACE] 12:56:06.115 shell.go:HasFilesInDir(host.json) - 0s
[TRACE] 12:56:06.115 shell.go:Pwd() - 0s
[TRACE] 12:56:06.115 shell.go:Pwd() - 0s
[DEBUG] 12:56:06.115 shell.go:HasFilesInDir:409 → false
[DEBUG] 12:56:06.115 shell.go:HasFilesInDir:409 → false
[TRACE] 12:56:06.115 shell_windows.go:Root() - 0s
[TRACE] 12:56:06.115 shell.go:HasFilesInDir(Gemfile) - 0s
[TRACE] 12:56:06.115 shell.go:HasFilesInDir(local.settings.json) - 0s
[TRACE] 12:56:06.115 shell.go:Pwd() - 0s
[DEBUG] 12:56:06.115 shell.go:HasFilesInDir:409 → false
[TRACE] 12:56:06.115 shell.go:HasFilesInDir(function.json) - 0s
[DEBUG] 12:56:06.116 shell.go:HasFilesInDir:409 → false
[TRACE] 12:56:06.116 shell.go:HasFilesInDir(*.py) - 1.0003ms
[DEBUG] 12:56:06.115 shell.go:FileContent:468 ↓
    [default]
    region = eu-west-3

[TRACE] 12:56:06.116 shell.go:FileContent(C:\Users\Bilge/.aws/config) - 1.0003ms
[TRACE] 12:56:06.116 shell.go:Pwd() - 0s
[DEBUG] 12:56:06.116 shell.go:HasFilesInDir:409 → false
[TRACE] 12:56:06.116 shell.go:HasFilesInDir(*.ipynb) - 0s
[TRACE] 12:56:06.116 shell.go:Pwd() - 0s
[DEBUG] 12:56:06.116 shell.go:HasFilesInDir:409 → false
[TRACE] 12:56:06.116 shell.go:HasFilesInDir(pyproject.toml) - 0s
[TRACE] 12:56:06.116 shell.go:Pwd() - 0s
[DEBUG] 12:56:06.116 shell.go:HasFilesInDir:409 → false
[TRACE] 12:56:06.116 shell.go:HasFilesInDir(venv.bak) - 0s
[DEBUG] 12:56:06.116 shell.go:HasFolder:438 → false
[TRACE] 12:56:06.116 shell.go:HasFolder(.venv) - 0s
[DEBUG] 12:56:06.116 shell.go:HasFolder:438 → false
[TRACE] 12:56:06.116 shell.go:HasFolder(venv) - 0s
[DEBUG] 12:56:06.116 shell.go:HasFolder:438 → false
[TRACE] 12:56:06.116 shell.go:HasFolder(virtualenv) - 0s
[DEBUG] 12:56:06.116 shell.go:HasFolder:438 → false
[TRACE] 12:56:06.116 shell.go:HasFolder(venv-win) - 0s
[DEBUG] 12:56:06.116 shell.go:HasFolder:438 → false
[TRACE] 12:56:06.116 shell.go:TemplateCache() - 0s
[TRACE] 12:56:06.116 shell.go:HasFolder(pyenv-win) - 0s
[DEBUG] 12:56:06.116 properties.go:GetBool:22 → always_enabled: true
[TRACE] 12:56:06.116 shell.go:TemplateCache() - 0s
[DEBUG] 12:56:06.116 text.go:Render:72 → Rendering template:  {{ if gt .Code 0 }}{{ reason .Code }}{{ else }}{{ end }}
[DEBUG] 12:56:06.116 text.go:Render:72 → Rendering template:  {{ .CurrentDate | date .Format }} 
[TRACE] 12:56:06.116 shell.go:TemplateCache() - 0s
[TRACE] 12:56:06.116 shell.go:TemplateCache() - 0s
[TRACE] 12:56:06.116 shell.go:Shell() - 0s
[TRACE] 12:56:06.116 shell.go:Shell() - 0s
[DEBUG] 12:56:06.116 text.go:Render:72 → Rendering template: powerline
[DEBUG] 12:56:06.116 text.go:Render:72 → Rendering template: powerline
[DEBUG] 12:56:06.116 text.go:Render:72 → Rendering template: powerline
[DEBUG] 12:56:06.116 text.go:Render:72 → Rendering template: powerline
[DEBUG] 12:56:06.116 text.go:Render:72 → Rendering template: powerline
[DEBUG] 12:56:06.116 text.go:Render:72 → Rendering template: powerline
[DEBUG] 12:56:06.116 text.go:Render:72 → Rendering template: powerline
[DEBUG] 12:56:06.116 text.go:Render:72 → Rendering template: powerline
[DEBUG] 12:56:06.116 text.go:Render:72 → Rendering template: {{ if gt .Code 0 }}#cc2222{{ end }}
[TRACE] 12:56:06.116 shell.go:TemplateCache() - 0s
[DEBUG] 12:56:06.116 text.go:Render:72 → Rendering template: powerline
[DEBUG] 12:56:06.116 text.go:Render:72 → Rendering template: diamond
[DEBUG] 12:56:06.117 shell_windows.go:TerminalWidth:111 → terminal width: 120
[TRACE] 12:56:06.117 shell_windows.go:TerminalWidth() - 999.6µs
[TRACE] 12:56:06.117 shell.go:Shell() - 0s
[TRACE] 12:56:06.117 shell.go:Shell() - 0s
[TRACE] 12:56:06.117 shell.go:Shell() - 0s
[TRACE] 12:56:06.117 shell.go:Shell() - 0s
[TRACE] 12:56:06.117 shell.go:Flags() - 0s
[TRACE] 12:56:06.117 shell.go:Pwd() - 0s
[DEBUG] 12:56:06.117 shell_windows.go:58 → C:\Users\Bilge
[TRACE] 12:56:06.117 shell.go:GOOS() - 0s
[DEBUG] 12:56:06.117 segment.go:SetEnabled:539 → Segment: Text
[TRACE] 12:56:06.117 shell.go:TemplateCache() - 0s
[DEBUG] 12:56:06.117 text.go:Render:72 → Rendering template: ❯
[TRACE] 12:56:06.117 shell.go:Shell() - 0s
[DEBUG] 12:56:06.117 shell.go:Getenv:326 → mintty
[TRACE] 12:56:06.117 shell.go:Getenv(TERM_PROGRAM) - 0s
[DEBUG] 12:56:06.117 text.go:Render:72 → Rendering template: plain
[TRACE] 12:56:06.117 shell.go:Shell() - 0s
[TRACE] 12:56:06.117 shell.go:Shell() - 0s
[TRACE] 12:56:06.117 shell.go:Flags() - 0s
[TRACE] 12:56:06.117 shell.go:Shell() - 0s
[TRACE] 12:56:06.117 shell.go:Shell() - 0s
[DEBUG] 12:56:06.117 shell.go:Getenv:326 → C:\Users\Bilge\AppData\Local
[TRACE] 12:56:06.117 shell.go:Getenv(LOCALAPPDATA) - 0s
[TRACE] 12:56:06.117 shell_windows.go:CachePath() - 0s
[TRACE] 12:56:06.117 shell.go:Flags() - 0s
```

--- a/src/shell/scripts/omp.bash
+++ b/src/shell/scripts/omp.bash
@@ -65,7 +65,7 @@ function _omp_hook() {
     set_poshcontext
     _set_posh_cursor_position
 
-    PS1="$(::OMP:: print primary --config="$POSH_THEME" --shell=bash --shell-version="$BASH_VERSION" --status="$ret" --pipestatus="${pipeStatus[*]}" --execution-time="$omp_elapsed" --stack-count="$omp_stack_count" --no-status="$no_exit_code" | tr -d '\0')"
+    PS1="$(::OMP:: print primary --config="$POSH_THEME" --shell=bash --shell-version="$BASH_VERSION" --status="$ret" --pipestatus="${pipeStatus[*]}" --execution-time="$omp_elapsed" --stack-count="$omp_stack_count" --no-status="$no_exit_code" --terminal-width="$COLUMNS" | tr -d '\0')"
     return $ret
 }
 

